tuxemon/core/npc.py

```diff
@@ -101,13 +101,24 @@ def __init__(self, npc_slug, sprite_name=None):
         # use 'animations' passed in
         # Hold on the the string so it can be sent over the network
         self.sprite_name = sprite_name
-
+        self.combat_front = combat_front;
+        self.combat_back = combat_back;
         if self.sprite_name is None:
             # Try to use the sprites defined in the JSON data
             try:
                 self.sprite_name = npc_data["sprite_name"]
             except KeyError:
                 logger.error('Cannot find sprite for {}'.format(npc_slug))
+        if self.combat_front is None:
```

Comment:
if these are None, then the combat state will not work at all.  in this case, since we are aware that this will cause an issue if there is None here we should let the game crash.  i would remove the try/except and let it crash with `KeyError`.  also, rewrite the `sprite_name` getter to be the same.  these are essential values and should always be filled.

Reply:
Makes sense.